### PR TITLE
My blog post, plus jquery lightbox

### PR DESCRIPTION
**Pull request by jmimico**
Please proof read the blog post. Chang helped implement jquery lightbox functionality as per my request.

### CODE REVIEW

**Comment by cheapsteak** [comment]
Deploy preview ready!

Built with commit dfb4f37b8921af3c44593a8f4a37c5c0d34242b2

https://deploy-preview-100--oicr-softeng.netlify.com

File: _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+5. [Allow SSH access to the instance](#5)
+6. [SSH to the instance](#6)
+7. [Install icgc-storage-client](#7)
+8. [Configure icgc-storage-client](#8)
+9. [Download ICGC data using icgc-storage-client](#9)
+
+<a id="1"></a>
+## Step 1: Log in to the Collaboratory OpenStack dashboard
+
+When you sign up for the Collaboratory you will receive some emails once your account is created.
```
Comment (Fgerthoffert) [inline _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md]: I would remove this section (i.e. the email account is a pre-requisite), eventually mentioning the email in the "Request Collaboratory account" bullet point above.

File: _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+
+In a browser, go to <https://console.cancercollaboratory.org> and log in using your provided credentials
+<figure>
+    <img id="console-login" src="{{site.urlimg}}jared_baker/day1collab/1-console-login.png" data-featherlight="#console-login" />
+</figure>
+
+The Collaboratory OpenStack dashboard is where you will manage your cloud environment. Creating things like instances (virtual machines, or vm's), volumes (additional storage), security groups (firewall rules), etc.
```
Comment (Fgerthoffert) [inline _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md]: Maybe make it just one sentence ?

File: _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+
+You can also view the detail of your keys and see the public key pair which is used on the instances you create and stored in the Collaboratory.
+
+<figure>
+    <img id="key-detail" src="{{site.urlimg}}jared_baker/day1collab/2-key-detail.png" data-featherlight="#key-detail" />
+</figure>
+
+<a id="3"></a>
+## Step 3: Create an instance
+Instances are Virtual Machines, aka VM's. Let's begin creating a single instance by clicking Launch Instance under the 'Instances' tab. This will launch an instance creation wizard where we will need to provide some additional information before an instance can be created.
```
Comment (Fgerthoffert) [inline _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md]: Lots of "instances", maybe replace the last one by "it"

**Review by Fgerthoffert — COMMENTED** [review]
I like that you added some explanations about some of the options "if you select this, it will do that".

File: _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+    image: jared_baker/day1collab/icgc-collab-splash.png
+header:
+    version: small
+    title: Software Engineering Blog
+    image: icgc-collab-splash.png
+    icon: icon-blog
+---
+
+## Introduction
+This is a tutorial meant to cover from signup to downloads of Cancer data at the Cancer Genome Collaboratory. Cancer researchers looking to perform analysis at the Collaboratory will need to do a number of things before they can start their analysis. I will walk you through each step to help get you on your way!
```
Comment (gmihaiescu) [inline _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md]: Maybe "This is a tutorial meant to cover the steps required to get from signup to download of Cancer data at the Cancer Genome Collaboratory" sounds better?

File: _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md [hunk 5]
```diff
@@ -0,0 +1,276 @@
...
+
+## Prerequisites
+* [Obtain DACO approval.](http://icgc.org/daco) This is a Collaboratory prerequisite and all users of the Collaboratory must have DACO approval. DACO handles requests from scientists for access to controlled data from the International Cancer Genome Consortium (ICGC).
+
+* [Request Collaboratory account](http://www.cancercollaboratory.org/services-request-account). The Cancer Genome Collaboratory is the Compute environment and also home to an object storage repository containing hundreds of terabytes of ICGC data. A new & easier signup page is coming soon. Upon signup, you will receive your account details via email.
+
+## Summary of Steps
+1. [Log in to the Collaboratory OpenStack dashboard](#1)
+2. [Create a key pair](#2)
```
Comment (gmihaiescu) [inline _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md]: "Create a SSH key pair" instead of "Create a key pair"

File: _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md [hunk 6]
```diff
@@ -0,0 +1,276 @@
...
+        <figcaption>SSH keys are used in favor of login/password authentication</figcaption>
+</figure>
+
+The key is downloaded by your browser as \<key-name>.pem. Make sure to restrict access to this file as it is your private key.
+
+~~~bash
+chmod 600 collaboratory-key.pem
+~~~
+
+Note, if you are on Windows and use Putty you will need to convert your .pem to .ppk.  [Follow these instructions](https://github.com/davidheijkamp/docs/wiki/Howto:-Creating-and-using-OpenStack---SSH-keypairs-on-Windows)
```
Comment (gmihaiescu) [inline _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md:71]: Linking to external github repo is risky, as they might not be there next week, but I guess a quick Internet search by any user can retrieve active instructions for using Putty with PEM keys.

File: _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md [hunk 7]
```diff
@@ -0,0 +1,276 @@
...
+
+You can create and manage your keys from the Access & Security tab.
+
+<figure>
+    <img id="key-list" src="{{site.urlimg}}jared_baker/day1collab/2-key-list.png" data-featherlight="#key-list" />
+</figure>
+
+You can also view the detail of your keys and see the public key pair which is used on the instances you create and stored in the Collaboratory.
```
Comment (gmihaiescu) [inline _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md]: "is used on the instances you create and stored in the Collaboratory" should be "is used on the instances you created and stored in the Collaboratory."

File: _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md [hunk 8]
```diff
@@ -0,0 +1,276 @@
...
+
+<figure>
+    <img id="key-detail" src="{{site.urlimg}}jared_baker/day1collab/2-key-detail.png" data-featherlight="#key-detail" />
+</figure>
+
+<a id="3"></a>
+## Step 3: Create an instance
+Instances are Virtual Machines, aka VM's. Let's begin creating one by clicking 'Launch Instance' under the 'Instances' tab. This will launch a wizard where we will need to provide some additional information before it can be created.
+
+Provide a useful name to identify the instance by. Leave the count at 1. If we wanted more instances like the one we are creating we could put the number desired here and your instances would be named dynamically. Example, if you instance name was 'my-first-instance' and the count was 3, you would have 3 instances with names 'my-first-instance-1, my-first-instance-2, my-first-instance-3'.
```
Comment (gmihaiescu) [inline _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md]: Switching between "we" and "you" makes it a bit hard to follow..

Comment (gmihaiescu) [inline _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md]: "Example, if you instance name" should be "Example, if your instance name"

File: _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md [hunk 9]
```diff
@@ -0,0 +1,276 @@
...
+
+<figure>
+    <img id="instance-name" src="{{site.urlimg}}jared_baker/day1collab/3-instance-name.png" data-featherlight="#instance-name" />
+</figure>
+
+The Source tab is where you will choose the Operating System that your instance will run. Ensure that your boot source is **'Image'** and not 'Volume' and that 'Create New Volume' is set to **'No'** This will make sure that your instance is provisioned correctly and will use the local disk of the underlying compute host. An instance backed by a Volume will have lower disk I/O performance and will incur extra charges.
```
Comment (gmihaiescu) [inline _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md]: Add the missing "." to the end of the second sentence.

File: _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md [hunk 10]
```diff
@@ -0,0 +1,276 @@
...
+
+<figure>
+    <img id="instance-source" src="{{site.urlimg}}jared_baker/day1collab/3-instance-source.png" data-featherlight="#instance-source"/>
+</figure>
+
+Instances in the Collaboratory come in different *flavours*. Flavours are what defines the specifications (# of CPU, RAM, and local disk space) of the instance. Choose a flavour to meet your computing needs.
```
Comment (gmihaiescu) [inline _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md]: Replace "Choose a flavour to meet your computing needs." maybe with "Choose an appropriate flavour to meet your computing needs while keeping in mind that you will have to terminate the instance and recreate it with a different flavour if you need a smaller or larger flavour. One other option would be to take a snapshot and start a new instance from it, but the flavour of the instance started from a snapshot cannot be smaller than the flavour of the instance initially being snapshotted."

File: _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md [hunk 11]
```diff
@@ -0,0 +1,276 @@
...
+    <img id="instance-flavor" src="{{site.urlimg}}jared_baker/day1collab/3-instance-flavor.png" data-featherlight="#instance-flavor" />
+</figure>
+
+Select a Network for your instance to communicate on. New projects already have a network created for them, but you can create more if needed.
+
+<figure>
+    <img id="instance-networks" src="{{site.urlimg}}jared_baker/day1collab/3-instance-networks.png" data-featherlight="#instance-networks"/>
+</figure>
+
+Security groups the firewall for your instance. They define what can and cant go to/from your Instance over the network. Choose the default security group for now as we will modify this later on.
```
Comment (gmihaiescu) [inline _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md]: "Security groups the firewall for your instance" -> what does it mean?
"go to/from your Instance over"  -> "Instance" should be in lower case

File: _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md [hunk 12]
```diff
@@ -0,0 +1,276 @@
...
+</figure>
+
+<a id="6"></a>
+## Step 6: SSH to the instance
+It's time to SSH to your instance.
+
+You will need:
+- The floating IP address of the instance
+- Your SSH private key (the .pem downloaded in step 2)
+- The username, which is standardized based on what OS/Image used (ubuntu, centos, debian)
```
Comment (gmihaiescu) [inline _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md]: Maybe add a note and a screenshot about the username for each base image being provided in the image description?

File: _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md [hunk 13]
```diff
@@ -0,0 +1,276 @@
...
+Copy your token ID and paste it in to the accessToken line in 'application.properties' in the 'conf' directory of the icgc-storage client. Save and exit!
+
+<figure>
+    <img id="icgc-appprop" src="{{site.urlimg}}jared_baker/day1collab/8-icgc-appprop.png" data-featherlight="#icgc-appprop" />
+</figure>
+
+<a id="9"></a>
+## Step 9: Download ICGC data using icgc-storage-client
+
+Using the [ICGC data portal](https://dcc.icgc.org) find the data of interest, ensuring its available in the Collaboratory using the built in filtering and copy the object ID
```
Comment (gmihaiescu) [inline _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md]: "Its available" should be "It's available"

File: _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md [hunk 14]
```diff
@@ -0,0 +1,276 @@
...
+
+<figure>
+    <img id="icgc-token" src="{{site.urlimg}}jared_baker/day1collab/8-icgc-tokenmgr.png" data-featherlight="#icgc-token" />
+</figure>
+
+<figure>
+    <img id="icgc-tokenlist" rc="{{site.urlimg}}jared_baker/day1collab/8-icgc-tokenlist.png" data-featherlight="#icgc-tokenlist" />
+</figure>
+
+Copy your token ID and paste it in to the accessToken line in 'application.properties' in the 'conf' directory of the icgc-storage client. Save and exit!
```
Comment (gmihaiescu) [inline _posts/jared_baker/2017-10-20-Day-1-Collaboratory.md]: Add a note about changing the "transport.memory" and "transport.parallel" to use x GB of RAM per CPU available, so if the instance has 8 cores and 48 GB of RAM, then use these settings while allowing one core and sufficient RAM for the operating system:

transport.memory=6
transport.parallel=7

This will allow 7 parallel download streams with each using 6 GB of RAM to improve the download speed.

**Comment by jmimico** [comment]
@gmihaiescu I have addressed all your critiques, thank you!

**Review by gmihaiescu — APPROVED** [review]
Looking good, merging.
